README.md

```diff
@@ -12,6 +12,12 @@ $ npm install
 $ npm run docs
 ```
 
+### Previewing
+1. We use [Yalc](https://github.com/whitecolor/yalc) to preview changes made to Molekule in consuming applications. Yalc can be installed by running: `yarn global add yalc`.
```

Comment:
Suggestion: Move first sentence to a paragraph under `Previewing` header as an introduction and update first step to `Install Yalc: `<command>` so the steps can be more concise and consistent